### PR TITLE
Time specificity UI

### PR DESCRIPTION
**Pull request by brmscheiner**

### CODE REVIEW

File: src/components/fields/view/SpecifiedTimeViewer.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { get } from 'lodash-es';
+
+import { formatDateCustom } from '../../../utils/formatters';
+import Text from '../../Text';
+
+const formatMap = {
```
Comment (Atticus29) [inline src/components/fields/view/SpecifiedTimeViewer.jsx:7]: It seems like you might could DRY this up by filtering on precisionMap from SpecifiedTimeEditor? As in, perhaps they could both rely on this similar object in a utils files instead?

File: src/constants/fieldTypesNew.js [hunk 2]
```diff
@@ -9,6 +9,7 @@ const fieldTypes = {
   latlong: 'latlong', // [float, float] defines [lat, lng]
   date: 'date',
   daterange: 'daterange', // [date, date]
+  specifiedTime: 'specifiedTime', // { time: string, timePrecision: string }
```
Comment (Atticus29) [inline src/constants/fieldTypesNew.js:12]: This file name tho.

**Review by Atticus29 — APPROVED** [review]
Approved with a few comments.

File: src/constants/fieldTypesNew.js (hunk 2 above)
Comment (brmscheiner) [reply src/constants/fieldTypesNew.js:12]: yeahh i still need to delete the old one!!!

Comment (brmscheiner) [reply src/constants/fieldTypesNew.js:12]: it's on my todo list...

File: src/components/fields/view/SpecifiedTimeViewer.jsx (hunk 1 above)
Comment (brmscheiner) [reply src/components/fields/view/SpecifiedTimeViewer.jsx]: good call - done!

**Pull request merged**